### PR TITLE
UNOMI-782: fix flaky test SegmentIT

### PR DESCRIPTION
**Pull request by jkevan**

### CODE REVIEW

**Review by jsinovassin — APPROVED** [review]

File: itests/src/test/java/org/apache/unomi/itests/SegmentIT.java [hunk 1]
```diff
@@ -531,8 +527,9 @@ public void testScoringRecalculation() throws Exception {
         pastEventCondition.setParameter("minimumEventCount", 1);
         pastEventCondition.setParameter("maximumEventCount", 2);
 
-        pastEventCondition.setParameter("fromDate","2000-07-15T07:00:00Z");
-        pastEventCondition.setParameter("toDate","2001-01-15T07:00:00Z");;
+        pastEventCondition.setParameter("fromDate", "2000-07-15T07:00:00Z");
+        pastEventCondition.setParameter("toDate", "2001-01-15T07:00:00Z");
+        ;
```
Comment (sergehuber) [inline itests/src/test/java/org/apache/unomi/itests/SegmentIT.java:532]: Small typo

Comment (sergehuber) [reply itests/src/test/java/org/apache/unomi/itests/SegmentIT.java:532]: We should just remove the ;

File: itests/src/test/java/org/apache/unomi/itests/SegmentIT.java [hunk 2]
```diff
@@ -547,41 +544,47 @@ public void testScoringRecalculation() throws Exception {
...
+            try {
+                boolean eventCounted = (Integer) ((Map) updatedProfile.getSystemProperties().get("pastEvents"))
+                        .get(pastEventCondition.getParameterValues().get("generatedPropertyKey")) == 2;
+                boolean profileEngaged = updatedProfile.getScores().containsKey("past-event-scoring-test")
+                        && updatedProfile.getScores().get("past-event-scoring-test") == 50;
+                return eventCounted && profileEngaged;
+            } catch (Exception e) {
+                // Do nothing, unable to read value
+            }
+            ;
```
Comment (sergehuber) [inline itests/src/test/java/org/apache/unomi/itests/SegmentIT.java:585]: Don't think we need this one do we ?

**Review by sergehuber — COMMENTED** [review]
Hello Kevan, 
Thanks for the fixes, I've just found some minor things that looks like some typos.
Regards,
  Serge...

**Comment by sergehuber** [comment]
The typos were corrected by Kevan separately in both master and this branch. Thanks !

**Pull request merged**